taskfiles/k8s.yml

```diff
@@ -28,6 +30,15 @@ tasks:
           rbac:roleName=manager-role \
           output:crd:artifacts:config=config/crd/bases \
           output:rbac:artifacts:config=config/rbac/bases/operator
+      - |
+        {{.K8S_APPLYCONFIGURATION_GEN_INSTALL_DIR}}/bin/applyconfiguration-gen \
+        --go-header-file "../../../licenses/boilerplate.go.txt" \
+        --output-dir "api/applyconfiguration" \
+        --output-pkg "github.com/redpanda-data/redpanda-operator/src/go/k8s/api/applyconfiguration" \
+        ./api/redpanda/v1alpha2
+      - find ./api/applyconfiguration/redpanda/v1alpha2 -type f -exec sed -i'' 's/"redpanda\/v1/"cluster.redpanda.com\/v1/g' {} \;
+      - rm -rf api/applyconfiguration/utils.go api/applyconfiguration/internal
```

Comment:
the applyconfiguration stuff also generates some `ForKind` and schema parsing helpers we don't need or use, so I just remove them as part of the generation step